sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestManager.java

```diff
@@ -722,6 +722,11 @@ private void messageReceived(final ChannelHandlerContext context, final RntbdRes
             return;
         }
 
+        if (response.getDecodeEndTime() != null && response.getDecodeStartTime() != null) {
```

Comment:
Shouldn't we have separate checks for these?
In case decode started, and never finished. It may provide insights on gaps in the SDK or decode process or any internal errors. Like we have for other events. where we see null end time.

Reply:
I forget why the specific checks were added previously.

But currently, the decode startTime and endTime will either both exists or both null.